src/common/options.cc

```diff
@@ -7570,6 +7570,11 @@ static std::vector<Option> get_rbd_options() {
     .set_min(1)
     .set_description("the number of quiesce notification attempts"),
 
+    Option("rbd_default_snapshot_quiesce_mode", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("obey")
+    .set_enum_allowed({"obey", "try", "skip"})
```

Comment:
Nit: I would use `require(d)` instead of `obey` and perhaps `ignore-error` instead of `try` (or alternatively expand the description)